src/udev/udev-event.c

```diff
@@ -583,14 +584,15 @@ static int on_spawn_sigchld(sd_event_source *s, const siginfo_t *si, void *userd
 
         switch (si->si_code) {
         case CLD_EXITED:
-                if (si->si_status != 0)
-                        log_warning("process '%s' failed with exit code %i.", spawn->cmd, si->si_status);
-                else {
+                if (si->si_status == 0) {
                         log_debug("process '%s' succeeded.", spawn->cmd);
                         sd_event_exit(sd_event_source_get_event(s), 0);
 
                         return 1;
-                }
+                } else if (spawn->accept_failure)
+                        log_debug("process '%s' failed with exit code %i.", spawn->cmd, si->si_status);
```

Comment:
This is english langauge, please uppercase the first character of a sentence. Thanks!